.gitignore

```diff
@@ -24,16 +24,16 @@ nuget_root/
 *.code-workspace
 __pycache__
 onnxruntime_profile*.json
-/docs/python/*.md
-/docs/python/auto_examples/*
-/docs/python/media/*
-/docs/python/examples/*.onnx
-/docs/python/examples/graph.*
+/docs/python/inference/*.md
```

Comment:
I'm not sure what depends on this, but can we avoid changing the path for inference docs?

(This seems to be related to a lot of file moves down below as well.)

Reply:
Unfortunately the move can't be avoided to keep a single docs folder.
The way sphinx works, it takes ownership of the entire root folder (docs in this case) folder and its subfolders. creating a docs/training subfolder would result in training doc going into inference folder

Future PRs might unify both docs in a single one, but this first PR only enables inference and training to coexist

Reply:
Can we rollback that change and propose it as a PR against master then? It seems like this is the only inference-affecting part of the PR, and we shouldn't hold up the rest of the merge on it.

Reply:
ps: the directory change had been approved by Pranav and Xavier Dupre (author of doc gen on Inference side)